package.json

```diff
@@ -32,8 +32,7 @@
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
     "express": "^4.12.0",
-    "loopback": "2.x.x",
-    "loopback-datasource-juggler": "2.x.x",
```

Comment:
You should look at the tests and determine whether they are being used or not. If they are being used, you need to determine whether they are candidates to refactor from using loopback to juggler directly.